Fix events (#548)
foundry-rs/starknet-foundry: docs/src/appendix/cheatcodes/spy_events.md

}
```

> ⚠️ **Warning**
>

> > Sharing contracts to spy on between spies can result in unexpected behavior — avoid it if possible.

Reviewer: ```suggestion
> Spying on the same contract with multiple spies can result in unexpected behavior — avoid it if possible.
```
I think it is clearer that way
